actions/setup-keeper/action.yml

```diff
@@ -11,14 +11,16 @@ runs:
       shell: bash
       run: pip3 install cryptography==37.0.2 pyOpenSSL==22.0.0 keepercommander==16.6.5
 
+    - uses: actions/github-script@v6
+      with:
+        script:
+          const config = `${{ inputs.config }}`;
```

Comment:
I'm surprised there isn't a way to access incoming inputs as variables in `github-script`?

Reply:
yeah I was also surprised, but composite actions don't get the ENV variables made for inputs automatically. I guess it makes usage more explicit?

Reply:
I wonder if you could use this: https://github.com/actions/github-script#use-env-as-input

so define an ENV for the config for this step and then access it via `process.env.KEEPER_CONFIG`.